lib/ibmdb.js

```diff
@@ -493,38 +515,43 @@ IBMDB.prototype.replaceOrCreate = IBMDB.prototype.replace =
       // Execution for replaceOrCreate requires running 3
       // separate SQL statements. The last depends on the
       // result of the first couple.
+      var where = {};
+      where[id] = data[id];
```

Comment:
Please see [this comment](https://github.com/strongloop/loopback-ibmdb/pull/7/files#r75055510)